src/Psalm/Internal/Type/ArrayType.php

```diff
@@ -35,8 +35,10 @@ public function __construct(Union $key, Union $value, bool $is_list, ?int $count
 
     /**
      * @return (
-     *     $type is TArrayKey ? self : (
```

Comment:
`TArrayKey` doesn't exist.

As soon as I fixed `TemplateInferredTypeReplacer.php`, the error was detected.